src/fshtypes/rules/InsertRule.ts

```diff
@@ -17,7 +17,15 @@ export class InsertRule extends Rule {
   private fshifyParameters(): string {
     return this.params
       .map(originalParam => {
-        return originalParam.replace(/\\/g, '\\\\').replace(/,/g, '\\,').replace(/\)/g, '\\)');
+        if (/(^\s+)|(\s+$)|[,\)]/g.test(originalParam)) {
```

Comment:
I don't think you need to escape `)` in a regex character class (e.g., inside `[` `]`). So this can be: `/(^\s+)|(\s+$)|[,)]/g`